.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: go
 go:
-  - 1.12
-  - tip
```

Comment:
Was it depending on tip for anything? I assume we are far past whatever it was if there was something anyway.

Reply:
Nope, just was something to test the latest releases. =)